go/protocol/keybase1/extras.go

```diff
@@ -511,9 +511,11 @@ func (t TeamID) AsUserOrTeam() UserOrTeamID {
 	return UserOrTeamID(t)
 }
 
+const ptrSize = 4 << (^uintptr(0) >> 63) // stolen from runtime/internal/sys
```

Comment:
Just one more thing:

`var _ cache.Measurable = TeamID{}`

Same for UID, KID, and UserOrTeamID

Reply:
I wanted to avoid importing `cache`, but I guess I could add it to the `_test.go` file and if it becomes a problem we can always move tests into a separate package.

Reply:
Sounds good, thanks Song!

Reply:
Hmm I don't think we are pulling in kbfs in client tests. Perhaps this is fine as we are using them in kbfs repo already?

Reply:
Sure, I'm fine with that.

Reply:
I'm gonna merge this as is for now, and if we care about having those check elsewhere we can add in a different PR.

Reply:
Yeah I forgot that we're in the client repo here. Don't worry about it. If anything we can do the checks in KBFS.